doc/source/user_guide/io.rst

```diff
@@ -2459,7 +2459,7 @@ as a string:
 .. ipython:: python
    :suppress:
```

Comment:
It's a bit strange to me that this part is suppressed while the `with open(file_path, "r")` below isn't, but I let that be.